walk-rpde.sh

```diff
@@ -17,7 +17,11 @@ set -- $1 -1
 while [ "$2" -ne "0" ]
 do
   page_padded=$(printf '%0*d\n' ${#max} $page)
-  set -- $(curl -sS "$1" | jq '.' | tee rpde-$page_padded.json | jq -r --arg base "$base" '$base + .next, (.items | length)')
+  set -- $(curl -L -sS "$1" | jq '.' | tee rpde-$page_padded.json | jq -r '.next, (.items | length)')
```

Comment:
This bash is blowing my mind @alzadude but okay

So does this mean `set -- abc def ghi` sets `$1` to `abc`, `$2` to `def` and `$3` to `ghi`?